mapviz_plugins/src/marker_plugin.cpp

```diff
@@ -298,23 +287,22 @@ namespace mapviz_plugins
         point.transformed_point = transform * (markerData.local_transform * point.point);
         markerData.points.push_back(point);
       }
-      else if (markerData.display_type == visualization_msgs::Marker::TEXT_VIEW_FACING)
+      else if (markerData.display_type == visualization_msgs::Marker::LINE_STRIP ||
+        markerData.display_type == visualization_msgs::Marker::LINE_LIST ||
+        markerData.display_type == visualization_msgs::Marker::CUBE_LIST ||
+        markerData.display_type == visualization_msgs::Marker::SPHERE_LIST ||
```

Comment:
This will also fix color for `SPHERE_LIST` after these changes plugin will support individual colors for spheres.